src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx

```diff
@@ -159,6 +165,33 @@ const DescriptionStep = ({
     });
   };
 
+  const handleNewlinesAsSoftline = (event, editorState) => {
+    const selection = editorState.getSelection();
+
+    if (selection.isCollapsed()) {
+      setEditorState(RichUtils.insertSoftNewline(editorState));
+    } else {
+      const content = editorState.getCurrentContent();
+      let newContent = Modifier.removeRange(content, selection, 'forward');
+      const newSelection = newContent.getSelectionAfter();
+      const block = newContent.getBlockForKey(newSelection.getStartKey());
+
+      newContent = Modifier.insertText(
+        newContent,
+        newSelection,
+        '\n',
+        block.getInlineStyleAt(newSelection.getStartOffset()),
+        null,
+      );
+
+      setEditorState(
+        EditorState.push(editorState, newContent, 'insert-fragment'),
+      );
+    }
+
```

Comment:
Replace `if else` structure with if and early return
```suggestion
    if (selection.isCollapsed()) {
      setEditorState(RichUtils.insertSoftNewline(editorState));
      return
    } 
      const content = editorState.getCurrentContent();
      let newContent = Modifier.removeRange(content, selection, 'forward');
      const newSelection = newContent.getSelectionAfter();
      const block = newContent.getBlockForKey(newSelection.getStartKey());

      newContent = Modifier.insertText(
        newContent,
        newSelection,
        '\n',
        block.getInlineStyleAt(newSelection.getStartOffset()),
        null,
      );

      setEditorState(
        EditorState.push(editorState, newContent, 'insert-fragment'),
      );
 

```